src/commands/synthetics/interfaces.ts

```diff
@@ -427,14 +430,21 @@ export interface BasicAuthCredentials {
   password: string
   username: string
 }
-export interface TriggerConfig {
+
+export interface RemoteTriggerConfig {
   // TODO SYNTH-12989: Clean up deprecated `config` in favor of `testOverrides`
   /** @deprecated This property is deprecated, please use `testOverrides` instead. */
   config?: UserConfigOverride
   testOverrides?: UserConfigOverride
   id: string
   suite?: string
 }
+export interface LocalTriggerConfig extends Omit<RemoteTriggerConfig, 'id'> {
+  /** Can be used to link to an existing remote test. */
```

Comment:
Would the id be a public id? If so, do we need it at the root? Otherwise, the `LocalTestDefinition` should allow for an optional `publicId`, no?
The backend is expected to return an error if a public id is sent outside of `localTestDefinition`, to avoid any confusion between ephemeral tests and main tests.